### DISCUSSION TITLE
On topic of lifting ban of summoning different types of elementals in same battle.

### DISCUSSION THREAD

**Discussion by Alucard648** [ihhub/fheroes2#8225] [Ideas]
HotA 1.7 did this. Why not doing this in fHeroes2?
https://www.youtube.com/watch?v=7EfaDyKzJ4Y

**Comment by ihhub** [comment]
We are not Hota and HoMM2 is not HoMM3. Elementals in homm3 are a part of a town, this is not the case for homm2. We don't need to blindly follow what other people do.

**Comment by oleg-derevenetz** [comment]
I have an impression that this restriction was implemented by the OG designers for a reason: namely, different types of elementals have immunities to different spells, so if you are able to spam elementals of different types without any restrictions, you will be able to more easily adapt to the set of spells that the enemy hero has. But that's just a guess. Anyway, if HotA developers want to alter the game behavior, It's their own business, but we don't have to do the same.

**Comment by Branikolog** [comment]
I also vote against this. It looks like adding 9th hero to the hero hiring pool.
Summoning a single type of elementals is a core game mechanics, preventing from abusing 5th level spell, if an opposite player has  not full set of available spells, mentioned by @oleg-derevenetz .
